A59-audit-logging.md

```diff
@@ -232,12 +281,249 @@ Again this is a subset of what users can technically do with xDS RBACs in the wa
 described in the earlier section. But we think this represents the most common
 use cases.
 
-### Language Specific APIs
+### Built-in logger types
+
+We plan to implement the stdout logger as a built-in logger type. The type is
+named `stdout_logger`. This logger will not support any configuration and it
+outputs log entries to stdout in JSON format.
+
+Here is the example JSON configuration in the xDS case (see [PR](https://github.com/envoyproxy/envoy/pull/26453)).
+
+```json
+{
+  "name": "user-defined-logger-name",
+  "typed_config": {
+    "@type": "envoy.extensions.rbac.audit_loggers.stream.v3.StdoutAuditLog",
+  }
+}
+```
+
+Here is the example in the authorization policy case.
+
+```json
+{
+  "name": "stdout_logger",
+}
+```
+
+Following is an example log entry.
 
-_This section is to be done once the language-agnostic APIs are more finalized._
+```json
+{"grpc_audit_log":{"timestamp":1649376211,"rpc_method":"/pkg.Service/Foo","principal":"spiffe://foo/user1","policy_name":"example_policy","matched_rule":"admin_access","authorized":true}}
+```
 
-_What is certain for now is we need to support third-party audit logger
-implementations registered by users._
+The timestamp, as the number of seconds from the Unix Epoch, is generated from
```

Comment:
Are we sure that seconds since the Unix epoch is the right time format to use here?  The output is JSON, so it's presumably meant to be a bit more user-friendly.  Maybe we should format this as a human-readable time string?  Or is that just going to open up problems due to (e.g.) locale-specific formatting?

Reply:
When timestamp was still a field in the audit context, unix seconds was given as an example. So I assumed it was preferred but I don't have any opinion at all. 

We could change to a different format, e.g., RFC3339, which seems to be recommended generally.

Reply:
cc @erm-g who's authoring the stdout logger in Go. We need to ensure the output is consistent across languages.

Reply:
@markdroth I don't have a strong opinion, but I think in most use cases the output will be fed to some aggregation pipeline for further analysis. So Unix epoch format is an ok default.

Reply:
I have checked with @ejona86 about using RFC3339 seconds and that seems fine. Aggregation pipelines should be able to process either format. 

I have changed the example here.